### PR TITLE
WW Energizing Elixir Resource gains (Chi calc'ed not shown)

### PR DESCRIPTION
**Pull request by terndrup**
Calculates the amount of energy and chi gained from the Energizing Elixir talent for Windwalker parses, using the energize event.

NOTE: The chi is calculated and available in the `chiGained` field but the engine is currently always returning a static amount of chi wasted on cast (incorrectly) and therefore it is not shown yet.

Coryn - Peak of Serenity

### CODE REVIEW

**Comment by rp4rk** [comment]
Can you fix the lint error in this, please?

File: src/Parser/Monk/Windwalker/Modules/Talents/EnergizingElixir.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  };
+  chiGained = 0;
+  energyGained = 0;
+  energyWasted = 0;
+  chiSaved = 0;
+  eeCasts = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.ENERGIZING_ELIXIR_TALENT
+      .id);
```
Comment (anom0ly) [inline src/Parser/Monk/Windwalker/Modules/Talents/EnergizingElixir.js]: Style item - But can you move this all to a single line?

File: src/Parser/Monk/Windwalker/Modules/Talents/EnergizingElixir.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        this.energyGained += event.resourceChange - event.waste;
+      }
+      if (event.resourceChangeType === ResourceTypes.CHI) {
+        this.chiWasted += event.waste;
+        this.chiGained += event.resourceChange - event.waste;
+      }
+    }
+  }
+
+  statistic() {
```
Comment (anom0ly) [inline src/Parser/Monk/Windwalker/Modules/Talents/EnergizingElixir.js]: For the statistics box, to keep this consistent with other specs, can you re-write it as the following:
```js
  statistic() {
    return (<StatisticBox icon={<SpellIcon id={SPELLS.ENERGIZING_ELIXIR_TALENT.id }/>}
      value={this.energyGained}
      label={(
        <dfn data-tip={`from ${this.eeCasts} Energizing Elixir Casts`}>
          Energy gained
        </dfn>
        )}
        />
      );
    }
  }
```

This just moves the casts portion to the tooltip.

Comment (terndrup) [reply src/Parser/Monk/Windwalker/Modules/Talents/EnergizingElixir.js]: Will do, thanks!

**Comment by terndrup** [comment]
Made the requested changes and updated Changelog 👍

**Pull request merged**